parity/run.rs

```diff
@@ -468,8 +469,8 @@ fn execute_impl<Cr, Rr>(cmd: RunCmd, logger: Arc<RotatingLogger>, on_client_rq:
 	// spin up event loop
 	let event_loop = EventLoop::spawn();
 
-	// fetch service
-	let fetch = fetch::Client::new().map_err(|e| format!("Error starting fetch client: {:?}", e))?;
+	let mut net_conf = cmd.net_conf;
+	let fetch = new_fetch_client(net_conf.https_proxy.as_ref())?;
```

Comment:
```new_fetch_client(cmd.net_conf.https_proxy.as_ref())``` should be sufficient here because `new_fetch_client()` returns a value not a reference

Then we can have smaller scope for the ```let mut net_conf``` binding (i.e, as it was before)